crates/solidity/src/tests/cli-tests/tests/common.test.ts

```diff
@@ -76,3 +76,57 @@ describe("Default run a command from the help", () => {
         expect(result.output).not.toMatch(/([Ee]rror|[Ww]arning|[Ff]ail)/i);
     });
 });
+
+describe("Run resolc with source debug information enabled", () => {
```

Comment:
Can we verify in this test that some of the expected debug info is in the actual final code blob? Doesn't need to be fancy, a string match or something we expect only to appear with debug info would be sufficient.